contents/handbook/people/onboarding.md

```diff
@@ -157,3 +157,17 @@ We use GitHub Projects to track the status of Issues in an easily viewable way.
 You can also assign an Issue to a specific person, and tag it with a relevant label - use these to help people filter more easily.
 
 Each Small Team has its own Project for tracking their Issues - [full list here](https://github.com/orgs/PostHog/projects). Most teams run two week sprints - as part of onboarding, you will be invited to the relevant planning meetings. 
+
+## 60 & 90 day Check-ins
+
+The first 3 months of your time at PostHog is a probationary period, which for the vast majority of people will be plain sailing. However, to make sure you are being supported correctly, and setup for success we have two check-ins to make sure everything is going well. 
+
+## 60 day Check-in 
+
+The 60 day check-in will be put in a new-starts manager's calendar as an all-day event by the Ops team. This serves as a reminder to the manager that their new report has been at the business for 60 days and there is 30 days left before their probation period ends. The assumed state here is that everything is going well, however it is a good chance for the manager to receive informal feedback from the report themselves on how things are going and if they need any more support. It is also a good chance to deliver any feedback the report might need to work on before the end of their probation, or just in general. This does not need to be a formal meeting, it can be part of an already scheduled 1-1 or asynchronous, however the manager best sees fit. 
+
+The manager should also give an informal update to their manager (and to the exec team, if their manager is not on the exec team) on how the new start is progressing. 
+
+## 90 day check-in 
+
+The 90 day check-in will be a more formal meeting put in the calendar by Ops and will include:- The new employee, their manager, relevant member of the exec team. Before the meeting the manager will have collected 360 feedback about the new employee and will be ready to give that to the employee. There will also be a formal discussion about passing probation and any feedback that the new employee can use to keep progressing at PostHog. 
```

Comment:
Feels bureaucratic and low value add, and either way 90 days is too late to make meaningful changes before probation is up. There's really two options here: Either the person is doing well, in which case the manager can give some casual encouragement, or the person is not doing well, in which case manager + exec needs to get involved at the 60 day mark.

Reply:
I guess the counter point to this and I don't feel very strongly about it, is that when a startup like ours relies heavily on a probationary period, it can be a really positive indicator of support from the company to take time to acknowledge that the person is being successful and given the exec team are the people who monitor performance, progression, changes etc I think delivering this information in an organised manner will have a positive impact on the individual. 

If you have a probationary period looming over you and it kind of just dissipatees with casual encouragement from a manager, it could be misleading if the people in charge of things like:- compensation, career progression, HR issues etc aren't involved. 

I think if we take the probationary period seriously from a negative performance, we should also do it from a positive performance pov. 

Again I don't feel very strongly about it.

---

If we don't believe the 90 day check-in is valuable are we deprecating it?

Reply:
We can fluff the language a bit so that it's not so formal, but through personal experience when I first started, I didn't realize how serious the 90-day probation period was (it's not the norm at least here in the US) so I do think there needs to be some sort of formality here so new hires have clear expectations of what they'll need to do to be successful.

Reply:
Echoing Tim,

> The 90 day check-in will be a more formal meeting put in the calendar by Ops and will include:- The new employee, their manager, relevant member of the exec team. Before the meeting the manager will have collected 360 feedback about the new employee and will be ready to give that to the employee.

This sounds very bureaucratic, especially the "collecting feedback from peers" part. First up, 360 feedback should only (predominately) be given live and in person, not collected and put in a binder. Second, why? I don't feel this step adds anything other than "fluff because calendar said so". It should be clear enough from the feedback _in the first 90 days_ if things are going good or not, including feedback you get from your peers. 

Don't get me wrong, having a more formal checkup end to the probation period at 90 days sounds good, but the above feels like "what are all the things we could do, let's put that in the handbook".

Reply:
yeah this is all fair feedback and it probably sounds a lot more bureaucratic than I intended. 

Ultimately the goal here is to improve employee experience, if we take the probationary period seriously we should acknowledge that. 

Collecting feedback is intended to be informal here (over slack, in a 1-1. Not some kind of form or document). 

on the point :-

 _"It should be clear enough from the feedback in the first 90 days if things are going good or not, including feedback you get from your peers."_ 
 
 It doesn't seem like it will be practical to hold a 360 feedback session around the time of every new starts 90 days, particularly as we hire more people, so I guess this is just trying to find an alternative on the basis that onboarding is a really critical time in setting up employees for success, so the manager puts in a little extra effort, as does the exec team member who is going to be responsiible for their career progression, compensation, HR issues etc.

All that being said - perhaps we lean into "Trust and feedback over process" and just remove the 90 day check in completely and keep the 60 day check in as the backstop for the manager flagging performance issues(ideally they are flagged ahead of that).

Reply:
Agree with scrapping the 90 days, but I've done the collecting feedback bit with people (off my own initiative) and it's been very useful - scheduling a one off 360 degree feedback session for a single person isn't practical, but I don't want the feedback to just come from me as a manager.

Reply:
I think it's great to have the 30, 60 and 90 day reminders in the calendar. They keep us all aware of the passage of time, and you develop your own "should be able to do X" role-specific milestones that you then go through in the next 1:1. Especially the 90 day "congrats, you made it" checkpoint is very useful as a milestone.

However, as a manager, if you present new or surprising information in a performance review, you're a bad manager. The weekly 1:1-s, the 3x week standups and other touchpoints leave plenty of time to give feedback. I hope it's also not a surprise to anyone that all managers periodically discuss their direct reports with their own manager, in an effort to get ahead of issues. 

Thus, if there's an issue, it will have already been dealt with. If you did make it to 90 days, 99% chance you're good. 

If you somehow made it to 90 days and you're not doing good, then you can bet all the bitcoin in the world, that your performance will be discussed, at length, no matter if the handbook says we should or shouldn't. Requiring everyone to then submit a 360 (even informally on slack) will also be a waste.

Thus, I'll keep stating that the point as written now, just adds bureaucratic fluff with no substance. Happy to propose alternative wording if that's useful.

Reply:
The meta point I'm trying to make is that handbook should be _descriptive_ of what we do, what works, etc. It should not be prescriptive with "things we wish people would start doing". We have a culture of highly autonomous adults who take pleasure in ignoring rules, and any kind of nannying through the handbook just means the handbook is out of date on day one.

Reply:
I don't think all managers agree with you on the meta point @mariusandra as a few were asking what was expected of them at these 30/60/90 day checkins. This wasn't a proactive attempt to 'nanny', but a response to requests from managers to clarify what was expected of them through this process. 

Maybe we can frame these as suggestions, but I think it's reasonable to provide some detail here? (Whatever we ultimately decide the specific advice should be, which I don't have a strong opinion about.)

Reply:
I have nothing against suggestions on what to do, for new managers asking for this, but those suggestions should be the best practices that others are doing... not things we wish people would start doing.

Reply:
As a new employee I would really appreciate feedback from my peers after my first 90 days. I think it would be really valuable to making sure I keep improving, until the next 360 feedback session. I wouldn't describe that as a waste. I believe this is already happening as well.

Reply:
my 2 🪙 
> 90 days is too late to make meaningful changes before probation is up ...
> ... in which case manager + exec needs to get involved at the 60 day mark.

This was the important part in Tim's comment for me. I feel like it's important for the manager of the person to gather 360 feedback before the 60 day 1:1 with the new hire (and 30 day 1:1 maybe too) & inform the exec team if there's a problem. Happy to provide more context in person.

Reply:
Ok it seems that actually the way things work right now are going well, particularly in flagging issues in the 30-60 day window, and there is no need for too much to change, what could be helpful is just documenting a few suggestions of how a manager can deliver a good employee experience when things are going well, in terms of delivering feedback. However, these won't be a prescriptive process as we don't really need that. 

Maybe we can start again on this...